apps/meteor/tests/end-to-end/api/oauthapps.ts

```diff
@@ -74,36 +77,38 @@ describe('[OAuthApps]', () => {
 					expect(res.body).to.have.property('success', true);
 					expect(res.body).to.have.property('oauthApp');
 					expect(res.body.oauthApp._id).to.be.equal('zapier');
-				})
-				.end(done);
+					expect(res.body.oauthApp).to.have.property('clientSecret');
+				});
 		});
-		it('should return a 403 Forbidden error when the user does not have the necessary permission by client id', (done) => {
-			void updatePermission('manage-oauth-apps', []).then(() => {
-				void request
-					.get(api('oauth-apps.get'))
-					.query({ clientId: 'zapier' })
-					.set(credentials)
-					.expect(403)
-					.expect((res) => {
-						expect(res.body).to.have.property('success', false);
-						expect(res.body.error).to.be.equal('unauthorized');
-					})
-					.end(done);
-			});
+		it('should return only non sensitive information if user does not have the permission to manage oauth apps when searching by clientId', async () => {
```

Comment:
add a couple of tests to verify you cannot pass anything other than strings on the params pls

Reply:
I believe there's no way to add such test case because, actually, you can passs something that is not a string. We're using **type coercion** on ajv
That means anything that is not a string will be parsed to a correspondent string whenever possible